### PR TITLE
add announcement for 2.2 release

### PR DESCRIPTION
**Pull request by corinnekrych**
to get ready for Monday 9th release announcement
@cvasilak @matzew mind to review?

### CODE REVIEW

File: _posts/2015-03-06-aerogear-ios-2.2.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+- [aerogear-ios-sync-client](https://github.com/aerogear/aerogear-ios-sync-client): the client uses sync engine and a network implementation. Our libraries is based on WebSockets to provide real-time sync.
+
+- [Jedi demo](https://github.com/aerogear/aerogear-ios-cookbook/tree/master/Jedi): Star war fan, rejoice in discovering our two Jedi hobbies demos (DiffMatchPatch and JsonPatch).
+      
+**2. House cleaning tasks:**
+
+- Move to cocoapods 0.36.0-rc1 for using iOS dynamic framework.
+
+- Deprecate [aerogear-ios-httpstub](https://github.com/aerogear/aerogear-ios-httpstub) in favour of its original Obj-C OHHTTPStubs. Read more on [mailing list thread "AeroGearHttpStub is gone"](http://aerogear-dev.1069024.n5.nabble.com/aerogear-dev-AeroGearHttpStub-is-gone-long-live-OHHTTPStubs-td11016.html) and our collaboration in maintaining the lib.
```
Comment (matzew) [inline _posts/2015-03-06-aerogear-ios-2.2.md]: let's add a link to OHHTTPStubs repo

**Comment by matzew** [comment]
I like the content of this blog post

**Comment by matzew** [comment]
Let's wait until we have the SERVER bits released and announced

**Comment by corinnekrych** [comment]
let's close this PR in favor of:
https://github.com/aerogear/aerogear.org/pull/551

**Pull request closed without merging**